cmd/integration/oidc_integration_test.go

```
@@ -190,24 +225,56 @@ func RunOIDCIntegrationTests(t *testing.T) {
 			cleanUpConjurAccount := prepareConjurAccount(account)
 			defer cleanUpConjurAccount()
 
-			setupAuthenticator(account, tc.serviceID, tc.providerURI, tc.clientID, tc.clientSecret, tc.claimMapping)
+			emptyFound := checkVariables(account, 
+				tc.oidcConnection.providerURI, 
+				tc.oidcConnection.clientID, 
+				tc.oidcConnection.clientSecret, 
+				tc.authnOidcConfig.serviceID, 
+				tc.authnOidcConfig.claimMapping)
+
+			if emptyFound {
+				return
+			}
```

What effect does an empty `return` have on the pass/fail status of the parent test case?

My guess is that the parent test case "passes" either way. Maybe `checkVariables` should call some assertions instead of returning a pass/fail boolean.